phase2/addon/.eslintignore

```diff
@@ -0,0 +1,2 @@
+data/js/*jquery*
```

Comment:
these files!  FUTURE put these in 'third-party' folder or such for easy finding :)  Or change all names to .min. version for easy regex.

Hint: .eslintignore must be at top level